docs/file-reference/index.md

```diff
@@ -43,14 +45,14 @@ commands:
 - <commandObject>
 ```
 
-| Key           | Type                                          | Required | Description                                                                       |
-|---------------|-----------------------------------------------|----------|-----------------------------------------------------------------------------------|
-| schemaVersion | string                                        | yes      | Schema version of devfile                                                         |
-| metadata      | [metadataObject](#metadataobject)             | no       | Metadata information that describes the project                                   |
-| projects      | array of [projectObject](#project-object)     | no       | List of projects that devfile will use when creating your development environment |
-| components    | array of [componentObject](#component-object) | no       | List of components to be used within your development environment                 |
-| commands      | array of [commandObject](#commandobject)      | no       | List of commands to be executed                                                   |
-
+| Key             | Type                                                    | Required | Description                                                             |
+|-----------------|---------------------------------------------------------|----------|-------------------------------------------------------------------------|
+| schemaVersion   | string                                                  | yes      | Schema version of devfile                                               |
+| metadata        | [metadataObject](#metadataobject)                       | no       | Metadata information that describes the project                         |
+| starterProjects | array of [starterProjectObject](#starterproject-object) | no       | List of starter projects that can be used to bootstrap new projects.    |
```

Comment:
remove period end of `List of starter projects that can be used to bootstrap new projects` sentence

Reply:
ya we are removing, I raised PR before the slack discussion, still working on it